docs/contribution_guide.rst

```diff
@@ -168,37 +168,20 @@ file.
 1. Check that the dashboard loads in the notebooks
 
 We have a
-`Azure DevOps Pipeline <https://dev.azure.com/responsibleai/fairlearn/_build?definitionId=48&_a=summary>`_
+`Azure DevOps Pipeline <https://dev.azure.com/responsibleai/fairlearn/_build?definitionId=60&_a=summary>`_
```

Comment:
This file has changed considerably in my PR. FYI

Reply:
OK.

Reply:
This file itself doesn't exist anymore in my PR so I've copied the changes from this file into my branch. Just FYI